### PR TITLE
Fix bootargs.cfg

### PR DESCRIPTION
**Pull request by davidcassany**
This commit ensures empty variables are evaluated as empty strings, as it was noted unquoted variables in conditional tests were not evaluated as empty strings leading to weird behavior on special corner cases.

### CODE REVIEW

File: packages/grub2/config/bootargs.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # TODO we could sanity check $partlabel is set here so we can error out before even attempting to boot
 set kernel=/boot/vmlinuz
-if [ ${label} == ${system_label} ]; then
+if [ "${label}" == "${system_label}" ]; then
```
Comment (davidcassany) [inline packages/grub2/config/bootargs.cfg:3]: I got an unexpected behavior when system_label was an empty string, the comparison was always returning true. Using quotes just fixed it.

**Review by frelon — APPROVED** [review]
LGTM
